docs/pages.jl

```diff
@@ -15,7 +15,7 @@ pages = Any[
         #"model_creation/parametric_stoichiometry.md",# Distributed parameters, rates, and initial conditions.
         # Loading and writing models to files.
         "model_creation/model_visualisation.md",
-        #"model_creation/network_analysis.md",
+        "model_creation/network_analysis.md",
```

Comment:
This one didn't actually generate any errors, so shouldn't actually have been commented out in the first place.